isisd/isis_spf.c

```diff
@@ -377,8 +377,21 @@ static struct isis_vertex *isis_spf_add2tent(struct isis_spftree *spftree,
 	char buff[VID2STR_BUFFER];
 #endif
 
-	assert(isis_find_vertex(&spftree->paths, id, vtype) == NULL);
-	assert(isis_find_vertex(&spftree->tents, id, vtype) == NULL);
+	vertex = isis_find_vertex(&spftree->paths, id, vtype);
+	if (vertex != NULL) {
```

Comment:
Is there any way you can give more information about this miss-configuration that would allow the operator to narrow down where this vertex is missconfigured?

Reply:
I have added the id and type of the existing vertex to the log, and also made it more explicit that there is likely a sysId collision with an established neighbor; let me know if you think this is sufficient, not really sure what else we could report